### ISSUE TITLE
Blasted Lands Audio repeating

### ISSUE THREAD

**Issue by scotty0100** [cmangos/issues#266]
When in Blasted lands you hear "I can't drink anymore yet" repeatedly. Confirmed by using ".tele blasted" and the audio is there.

**Comment by cala** [comment]
Gentlemen,

I made yesterday several backports for Classic client and during one of them I found a malformed packet sent to game client. This is the packet related to the **sound played** for the weather in a zone by the client. Previously, the server was sending a packet shorter than what the client expects.

I remember someone saying that the looping sounds issue discussed here was related to zones without weather (like Blasted Lands, WG or Darnassus...). Unfortunately, I don't remember if this was on the forum or on IRC. Thus, I suggest that someone tests if the issue still occurs after https://github.com/cmangos/mangos-classic/commit/7593dd659a014de36e6823e970cca096ad402565 ?

I would gladly test this myself, but the client I use (Mac version) seems to be unaffected by this issue.
